packages/cms/form/select/select.tsx

```diff
@@ -1,31 +1,35 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { FormComponentProps } from '@tensei/components'
-import { EuiSelect } from '@tensei/eui/lib/components/form'
+import { EuiSuperSelect } from '@tensei/eui/lib/components/form'
 
 const FormSelect: React.FC<FormComponentProps> = ({
   field,
   name,
   id,
-  value,
-  onChange,
   onFocus,
   error
 }) => {
-  const options = field?.selectOptions?.map(option => {
-    return { text: option.label, value: option.value }
-  })
+  const options =
+    field?.selectOptions?.map(option => {
+      return { value: option.label, inputDisplay: option.value }
+    }) || []
+  const [value, setValue] = useState(options[1].value)
```

Comment:
You shouldn't be setting state. This component receives `value` and `onChange` already. Just pass them accordingly to this component.